.github/pull_request_template.md

```diff
@@ -0,0 +1,44 @@
+ADO card: ☑️ [AB#XXXX](https://dev.azure.com/S72/SHIFT72/_boards/board/t/Delivery%20team/Stories/?workitem=XXXX)
```

Comment:
Nit: This opens the board and then the card, prefer a direct link to the card. Also, I think we can just `AB#XXXX` for it to autolink.

Reply:
Direct link to card added.
AB#XXXX will autolink from the ADO card to Github, but to link from Github to the ADO card we need to put in the actual link.

Reply:
Hmm, go with this for now, but it's that the wrong way around. [GitHub auto links AB#XXXX within PR titles/descriptions](https://docs.microsoft.com/en-us/azure/devops/boards/github/link-to-from-github?view=azure-devops#use-ab-mention-to-link-from-github-to-azure-boards-work-items).

And, it used to work fine:

![image](https://user-images.githubusercontent.com/2253814/169176373-251fa034-aca4-48e5-8029-781098df9415.png)
![image](https://user-images.githubusercontent.com/2253814/169176378-ac77f25f-4d3d-44f9-89f3-d34d3d24c558.png)

But looks like someone has broke our integration 😱 - will follow up in Slack.